client/src/js/services/PatientInvoiceService.js

```diff
@@ -79,6 +82,34 @@ function PatientInvoiceService($http, util, Session) {
       .then(util.unwrapHttpResponse);
   }
 
+  /**
+   * @method search
+   *
+   * @description
+   * This method is reponsible of seaching for invoice(s) based on passed parameters
+   *
+   * @returns {Promise} - a promise resolving to the HTTP result.
+   */
+
+  function search (options) {
+    var target = baseUrl.concat('search');
+
+    /*
+     * Convertion of billingDateFrom and billingDateTo
+     */
+
+    if (options.billingDateFrom) {
```

Comment:
I know that the patient search did this, but there isn't actually any conversion necessary on the client for dates.  This will be done in the database.